docs/examples.md

```diff
@@ -7,6 +7,7 @@ sidebar_label: Resources
 There are a variety of single-spa example repositories, each for different use cases:
 - [coexisting-angular-microfrontends](https://github.com/joeldenning/coexisting-angular-microfrontends) is a full blown Angular 7 microfrontends repo that combines three separate Angular CLI projects into one page.
 - [coexisting-vue-microfrontends](https://github.com/joeldenning/coexisting-vue-microfrontends) shows three separate Vue CLI projects existing within one page.
+- [single-spa-vue-cli-3](https://github.com/ghulamMustafaRaza/single-spa-vue-cli-3) demonstrates multiple vueSPAs in sing vue cli project with single VueX store. 
```

Comment:
The coexisting-vue-microfrontends repo is using vue-cli 3 (see [here](https://github.com/joeldenning/coexisting-vue-microfrontends/blob/master/app1/package.json#L17)). Is the difference between coexisting-vue-microfrontends and single-spa-vue-cli-3 whether you have many single-spa apps within the same vue-cli project vs one vue-cli project per app?

I actually prefer encouraging a separate vue cli project for each single-spa application, since that enables independent deployability, separate git repos, etc for each of the apps.

Also, typo: `in sing vue cli project`